cpp/src/IceGrid/Activator.cpp

```diff
@@ -641,20 +641,50 @@ Activator::activate(const string& name,
     }
 
     vector<gid_t> groups;
+#ifdef _AIX
+    char* grouplist = getgrset(pw->pw_name);
+    if(grouplist == 0)
+    {
+        throw SyscallException(__FILE__, __LINE__, getSystemErrno());
+    }
+    vector<string> grps;
+    if(IceUtilInternal::splitString(grouplist, ",", grps))
+    {
+        for(vector<string>::const_iterator p = grps.begin(); p != grps.end(); ++p)
+        {
+            gid_t group;
+            istringstream is(*p);
+            if(is >> group)
+            {
+                groups.push_back(group);
+            }
+        }
+    }
+#else
     groups.resize(20);
     int ngroups = static_cast<int>(groups.size());
-#if defined(__APPLE__)
+    #if defined(__APPLE__)
```

Comment:
the indentation of this defines isn't correct

Reply:
I don't remember indenting these macros... Bernard?

Reply:
You're right, neither the original nor the latest version is correct. The line should start with # followed by a number of spaces for proper indentation. See IceUtil/Config.h for examples.

Reply:
fixed